#request control: 1
#release control: 0
uint8 control_enable
---
bool result
uint8 cmd_set
uint8 cmd_id
uint32 ack_data